docs/user/installing/installing-service-on-linux.md

```
@@ -172,19 +174,84 @@ On first startup, certificates are automatically generated with the following be
+(Optional) If services were previously run and derived certificates generated, remove them.
+
+> [!WARNING]
+> This is a destructive action.  Doing so will result in previously issued certificates becoming unverifiable and devices must be re-enrolled.
+
+```bash
+sudo rm -r /etc/flightctl/pki/*
+```
+
+Place the existing CA key and cert files in the following locations, also ensuring they are readable:
+
+  - `/etc/flightctl/pki/ca.crt`
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 0; Actual: 2

(MD007, ul-indent)